pkg/resource/aws/aws_iam_policy_attachment_ext.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+)
+
+func (r *AwsIamPolicyAttachment) NormalizeForState() (resource.Resource, error) {
+	return r, nil
+}
+
+func (r *AwsIamPolicyAttachment) NormalizeForProvider() (resource.Resource, error) {
+	if r.Groups != nil && len(*r.Groups) == 0 {
```

Comment:
This should be done in `NormalizeForState` too, if we have an empty slice in the state you'll introduced a new false positive

Reply:
Oh right,
I was thinking that this modification should not be made because it introduces a lot of problems in the tests which seemed to me unsuitable for the ticket, but I'll take a look at it again and solve them

Reply:
Should be ok now